site/docs/uk/hosting/vps.md

```diff
@@ -52,130 +152,631 @@ systemd --- це потужний менеджер служб, який пере
 2. Відкрийте ваш новий файл служби за допомогою редактора:
 
    ```sh
-   nano bot1.service
+   nano <назва-застосунку>.service
    ```
 
+   > Замініть `<назва-застосунку>` будь-яким ідентифікатором.
+   > `<назва-застосунку>.service` буде назвою вашого сервісу.
+
 3. Додайте наступний вміст:
 
-   ```txt
+   ```text
+   [Unit]
+   After=network.target
+
    [Service]
+   Environment=BOT_TOKEN=<токен-бота>
    ExecStart=<команда-запуску>
    Restart=on-failure
 
    [Install]
    WantedBy=multi-user.target
    ```
 
-   > Замініть `<команда-запуску>` на команду, яку ви отримали вище.
-   > Також зверніть увагу, що якщо Deno встановлено для іншого користувача, а не root, вам може знадобитися додати `User=<імʼя-користувача>` в розділі `[Service]`.
-   > Для отримання додаткової інформації про файли служб відвідайте [цей сайт](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_systemd_unit_files_to_customize_and_optimize_your_system/assembly_working-with-systemd-unit-files_working-with-systemd).
+   > Замініть `<токен-бота>` на токен вашого бота, а `<команда-запуску>` на команду, яку ви отримали [вище](#отримання-команди-запуску).
+
+   Ось коротке пояснення конфігурації сервісу:
+
+   - `After=network.target` --- вказує на те, що додаток повинен запускатися після завантаження інтернет-модуля.
+   - `Environment=BOT_TOKEN=<токен-бота>` --- встановлює змінну оточення `BOT_TOKEN`.
+     Додайте інші записи `Environment`, якщо вам потрібно декілька змінних оточення.
+   - `ExecStart=<команда-запуску>` --- встановлює команду запуску.
+   - `Restart=on-failure` --- вказує на те, що програма повинна перезапускатися після збою.
+   - `WantedBy=multi-user.target` --- визначає стан системи, в якому повинен запускатися сервіс.
+     `multi-user.target` --- типове значення для серверів, яке визначає стан системи, коли працює все, окрім графічного інтерфейсу, якого просто не існує.
```

Comment:
```suggestion
     `multi-user.target` --- типове значення для серверів, яке визначає стан системи.
```

Reply:
Do you think you can leave a user without this information like that? In this case, you should remove this part from the original as well.

Reply:
This information has absolutely no impact on the current deployment process. The user can also find out this information by opening the link we provided earlier.